sklearn/linear_model/_coordinate_descent.py

```diff
@@ -516,7 +552,7 @@ def enet_path(
     See Also
     --------
     MultiTaskElasticNet : Multi-task ElasticNet model trained with L1/L2 mixed-norm \
-    as regularizer.
+        as regularizer.
```

Comment:
This indentation seems unavoidable because the `sphinx` extension `numpydoc` raises an error message,and `sphinx` would not output the HTML files otherwise.